fhttp/http_client.go

```diff
@@ -308,6 +310,7 @@ type Client struct {
 	bodyContainsUUID     bool // if body contains the "{uuid}" pattern (lowercase)
 	logErrors            bool
 	id                   int
+	socketCount          *int
```

Comment:
why is it a pointer?

Reply:
I guess because the client is created after the transport but could either swap the order / have the initial client sooner or possibly have the pointer be only inside the transport

Reply:
so yes let's use an int and just swap most of the init order, making the client first and then assinging the .client and .transport components

Reply:
updated

Reply:
thx